lib/logstash/filters/fingerprint.rb

```diff
@@ -92,6 +97,16 @@ def register
         )
       end
       class << self; alias_method :fingerprint, :fingerprint_ipv4_network; end
+    when :IPV6_NETWORK
```

Comment:
As the implementation is the same as IPv4, I suggest to reuse the condition in ipv4 instead of duplicate the code
`when :IPV4_NETWORK, :IPV6_NETWORK`